internal/renderer/helm.go

```diff
@@ -38,6 +38,11 @@ post-process the serialized yaml to replace all 'template_'-prefixed keys
 with the appropriate content (value or template)
 */
 
+const (
+	// based on https://github.com/docker/compose/blob/1.21.2/compose/config/interpolation.py#L93-L104
+	envVarRegex = `(^|[^$])\$(?:(?P<named>[_a-zA-Z][_a-zA-Z0-9.]*)|{(?P<braced>[_a-zA-Z][_a-zA-Z0-9.]*)(?:(?P<sep>:?[-?])[^}]*)?})`
```

Comment:
It's the same as https://github.com/docker/app/pull/244/files#diff-805075bc7226dea804858742c65b1dabR241 :+1:

Reply:
http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_05_01 describes the syntax